guides/getting_started_command_line.rst

```diff
@@ -29,7 +29,7 @@ an RPi, please install ESPHome through ``pip`` or use :doc:`the Hass.io add-on <
 
     docker pull esphome/esphome
 
-Creating A Project
+Creating a project
```

Comment:
```suggestion
Creating a Project
```

Usually, the subtitles are in title-case (though I do not actively enforce that)